onebusaway-api-webapp/src/main/java/org/onebusaway/api/impl/TripsConfigurationServiceClientFileImpl.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (C) 2015 Cambridge Systematics, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.onebusaway.api.impl;
+
+import java.io.File;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.io.FileUtils;
+import org.codehaus.jackson.JsonFactory;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.type.TypeReference;
+import org.onebusaway.api.services.TripsConfigurationServiceClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonObject;
+
+public class TripsConfigurationServiceClientFileImpl implements
+		TripsConfigurationServiceClient {
+
+	private static Logger _log = LoggerFactory
+			.getLogger(TripsConfigurationServiceClientFileImpl.class);
+	private HashMap<String, Object> _config = null;
+	
+	private boolean isLocal = true;
+	
+	private String configFile = "/var/lib/oba/config.json";
```

Comment:
If you are using the existing convention of config.json, why build a new ConfigurationServiceClientFileImpl?  At the very least, you could have extended the existing FileImpl.